sources/packages/web/src/views/Login.vue

```diff
@@ -7,15 +7,22 @@
       <div>
         <h1>Welcome to Student Aid Portal</h1>
         <h4>
-          We are using BCSC for authentication. Please click on login button
-          below to start your sign up/sign in.
+          We are using BCSC for authentication. Please click on Login/Register buttons
+          below to start your sign in/sign up.
         </h4>
       </div>
     </template>
     <template #footer>
       <Button
-        label="Login using BCSC"
+        label="Login"
+        icon="pi pi-check"
+        class="p-mr-2"
```

Comment:
p-mr-2, mr is for medium sized devices right? should we worry about medium sized devices right now?.

Reply:
**mr** means "margin-right" here. It is adding spacing between the buttons as described in PrimeVue doc below:
https://www.primefaces.org/primevue/showcase/#/spacing

The below ones are the ones that you probably would be thinking about:

- sm: small screens e.g. phones
- md: medium screens e.g. tablets
- lg: large screens e.g. notebooks
- xl: larger screens .e.g monitors